Dont require method loading for meta meta in module api

JacobSzwejbka opened the pull request:
Summary: Program load is still required but this is very cheap. Method load is not needed.

Differential Revision: D76278890

shoumikhin commented:
Lgtm as long as all tests pass.
Maybe add CI/flow label to trigger everything.
And iirc there are some docs and doc comments to amend a bit

JacobSzwejbka committed:
Dont require method loading for meta meta in module api (#11496)

Summary:

Program load is still required but this is very cheap. Method load is not needed.

Differential Revision: D76278890
